Packs/DomainToolsIrisDetect/IndicatorFields/incidentfield-DomainTools_Iris_Detect_domaintoolsriskscore.json

```
@@ -0,0 +1,31 @@
+{
+    "id": "indicator_domaintoolsriskscore",
+    "version": -1,
+    "modified": "2023-01-24T16:23:01.084923322+05:30",
+    "name": "DomainTools Risk Score",
+    "ownerOnly": false,
+    "cliName": "domaintoolsriskscore",
+    "type": "shortText",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "openEnded": false,
+    "associatedTypes": [
+        "DomainTools Iris Detect "
```

```suggestion
        "DomainTools Iris Detect"
```
Please remove this extra space in all the other indicator fields as well.